### PR TITLE
Fix titles and content in Sawtooth documentation

### PR DESCRIPTION
**Pull request by chenette**
- Fix wording and capitalization for several titles (and remove ! from "Transactions and Batches")
- Correct a few wording problems, such as capitalization for the Identity and Settings transaction family 
- Change "CLI" to "command" throughout

### CODE REVIEW

File: docs/source/introduction.rst [hunk 1]
```diff
@@ -116,14 +116,14 @@ applications to:
...
-Ethereum contract compatibility with Seth
+Ethereum Contract Compatibility with Seth
 -----------------------------------------
 
 The Sawtooth-Ethereum integration project, Seth, extends the interoperability
 of the Sawtooth platform to Ethereum. EVM (Ethereum Virtual Machine) smart
 contracts can be deployed to Sawtooth using the Seth transaction family.
 
-Pluggable consensus algorithms
+Pluggable Consensus Algorithms
```
[the remark this answers is not in the document]
Comment (chenette) [reply docs/source/introduction.rst]: Yes!  I just saw this term in today's Hyperledger Sawtooth blog post about the 1.0 release. I shall replace "Pluggable" with "Dynamic".

[the remark this answers is not in the document]
Comment (chenette) [reply docs/source/introduction.rst]: Fixed.

File: docs/source/_templates/sdk_TP_tutorial.rst [hunk 2]
```diff
@@ -487,7 +487,7 @@ sophisticated, `BSON <http://bsonspec.org/>`_.
 Implementing Game Play
 ======================
 
-Next, describe how to play tic-tac-toe. The details here are fairly
+This tutorial does not include example code for playing tic-tac-toe. The details are fairly
```
Comment (dcmiddle) [inline docs/source/_templates/sdk_TP_tutorial.rst]: I don't think the added text is relevant/accurate. In the previous section there are the CLI commands for playing the game. I'd delete this text up through "straightforward, and". Just begin with "The functionality could certainly ..."

File: docs/source/app_developers_guide/aws.rst [hunk 3]
```diff
@@ -73,13 +73,13 @@ to log in to the instance you just launched. Use the username
 
 Once you've successfully logged in, continue to the next section.
 
-Using Sawtooth with the CLI Commands
-====================================
+Using Sawtooth Commands
```
Comment (dcmiddle) [inline docs/source/app_developers_guide/aws.rst]: I prefer: "Using Sawtooth Command Line Tools"
as `intkey` is a separate CLI than `sawtooth`
Kind of a nit though.

File: docs/source/app_developers_guide/docker.rst [hunk 4]
```diff
@@ -264,13 +264,13 @@ If the validator process or the validator container is not running, the ``curl``
 command will time out or return nothing.
 
 
-Using Sawtooth with CLI Commands
-================================
+Using Sawtooth Commands
```
Comment (dcmiddle) [inline docs/source/app_developers_guide/docker.rst]: as above in the aws.rst: prefer, "Using Sawtooth Command Line Tools"

File: docs/source/_templates/sdk_TP_tutorial.rst (hunk 2 above)
Comment (chenette) [reply docs/source/_templates/sdk_TP_tutorial.rst]: This section will be rewritten soon.  In the meantime, I'll delete the first few sentences and start with "The functionality could certainly...".

File: docs/source/app_developers_guide/aws.rst (hunk 3 above)
Comment (chenette) [reply docs/source/app_developers_guide/aws.rst]: The CLI / command / tool terminology is nonstandard and inconsistent across the Sawtooth doc set. I am making the process of making it consistent as I edit each section.  This change matches an earlier change to the --help text and the CLI Command Reference.  

FYI, I am following the model of Red Hat, Amazon AWS, and other docs. In brief, "CLI" is the umbrella term for the entire set of Sawtooth commands; "command" is the term for the first item you type on a command line, and "subcommand" is the 2nd or 3rd-level item (before options and arguments) for a command, as in "sawtooth batch" or "sawnet peers list".

**Pull request merged**